unit/util/irep.cpp

```diff
@@ -145,7 +143,7 @@ SCENARIO("irept_memory", "[core][utils][irept]")
 
       irep.add("#a_comment", irep2);
       REQUIRE(irep.find("#a_comment").id() == "second_irep");
-      REQUIRE(irep.get_comments().size() == 1);
+      REQUIRE(irep.get_named_sub().size() == 2);
```

Comment:
We should now include `is_comment` in this unit test (the test aims to cover the full API).